src/api/specimen/GetRecentSpecimens.ts

```diff
@@ -0,0 +1,26 @@
+/* Import Dependencies */
+import axios from "axios";
+
+/* Import Types */
+import { Specimen } from "global/Types";
+
+
+const GetRecentSpecimens = async () => {
+    let recentSpecimens = <Specimen[]>[];
+
+    const endPoint = "/specimens"
+
+    axios({
```

Comment:
Here you don't use the await I think this is how it should look

Reply:
It is not quite the same because await really waits for the call to complete before anything in the function can be executed, .then does execute when the result is reeled in, but the other stuff in the function will continue. Chose await over .then for the API calls because of the waiting, think this is important. Removed .then for all api calls and introduced try/catch for checking if the fetch was successful or not.